src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -150,7 +150,10 @@ def test_delete_failure_not_found(resource_client, current_resource_model):
 def test_input_equals_output(resource_client, input_model, output_model):
     pruned_input_model = prune_properties_from_model(
         input_model.copy(),
-        list(resource_client.write_only_paths, resource_client.read_only_paths),
```

Comment:
pre-commit works for some reason. I think it is because we don'gt have unit test for handler_common. Might be worth adding testing to make sure this is caught in pre-commit